Test suite failing when manager loads with no sequence paths

* Command Sequencer version: master
* Python version: 3.9
* Operating System: Any

### Description

Confirming test suite passes on master before reviewing and merging recent PR. Multiple test cases fail with type error when instantiating a manager with without sequence file paths

### What I Did

```
$ pytest
======================================================================== test session starts ========================================================================
platform linux -- Python 3.9.7, pytest-7.1.2, pluggy-1.0.0
rootdir: /aeg_sw/work/users/tcn45/develop/projects/odin/odin-sequencer, configfile: setup.cfg
plugins: datadir-1.3.1, cov-3.0.0
collected 144 items

test_adapter.py EEEEEE                                                                                                                                        [  4%]
test_command_sequencer.py F..F.F..F.F..F....F..........FFFF..F                                                                                                [ 29%]
test_file_watcher_factory.py ........                                                                                                                         [ 34%]
test_inotify_file_watcher.py ..................                                                                                                               [ 47%]
test_odin_sequencer.py ..........................................................                                                                             [ 87%]
test_standalone_file_watcher.py ..................                                                                                                            [100%]

============================================================================== ERRORS ===============================================================================
______________________________________________ ERROR at setup of TestCommandSequenceManagerAdapter.test_get_valid_path ______________________________________________

cls = <class 'tests.test_adapter.TestCommandSequenceManagerAdapter'>

    @classmethod
    def setup_class(cls):
>       cls.adapter = CommandSequenceManagerAdapter()

test_adapter.py:32:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
../src/odin_sequencer/adapter.py:34: in __init__
    self.command_sequencer = CommandSequencer(sequence_location)
../src/odin_sequencer/command_sequencer.py:52: in __init__
    self._initialise_manager()
../src/odin_sequencer/command_sequencer.py:64: in _initialise_manager
    self.manager.load(self.initial_path_or_paths)
../src/odin_sequencer/manager.py:81: in load
    path = Path(path)
/aeg_sw/tools/Ubuntu20-x86_64/python/3-9-7/prefix/lib/python3.9/pathlib.py:1082: in __new__
    self = cls._from_parts(args, init=False)
/aeg_sw/tools/Ubuntu20-x86_64/python/3-9-7/prefix/lib/python3.9/pathlib.py:707: in _from_parts
    drv, root, parts = self._parse_args(args)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

cls = <class 'pathlib.PosixPath'>, args = (None,)

    @classmethod
    def _parse_args(cls, args):
        # This is useful when you don't want to create an instance, just
        # canonicalize some constructor arguments.
        parts = []
        for a in args:
            if isinstance(a, PurePath):
                parts += a._parts
            else:
>               a = os.fspath(a)
E               TypeError: expected str, bytes or os.PathLike object, not NoneType

<<snip>>

========================================================================= warnings summary ==========================================================================
../../../../../../../../../u/tcn45/.virtualenvs/odin-sequencer-ubuntu-3.9/lib/python3.9/site-packages/_pytest/config/__init__.py:1252
  /u/tcn45/.virtualenvs/odin-sequencer-ubuntu-3.9/lib/python3.9/site-packages/_pytest/config/__init__.py:1252: PytestConfigWarning: Unknown config option: collect_ignore

    self._warn_or_fail_if_strict(f"Unknown config option: {key}\n")

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
====================================================================== short test summary info ======================================================================
FAILED test_command_sequencer.py::test_command_sequencer_with_no_paths - TypeError: expected str, bytes or os.PathLike object, not NoneType
FAILED test_command_sequencer.py::test_get_missing_param - TypeError: expected str, bytes or os.PathLike object, not NoneType
FAILED test_command_sequencer.py::test_set_missing_param - TypeError: expected str, bytes or os.PathLike object, not NoneType
FAILED test_command_sequencer.py::test_set_detect_module_modifications_to_true_when_no_modules_loaded - TypeError: expected str, bytes or os.PathLike object, not ...
FAILED test_command_sequencer.py::test_set_detect_module_modifications_to_false_when_not_enabled - TypeError: expected str, bytes or os.PathLike object, not NoneType
FAILED test_command_sequencer.py::test_module_modifications_detected_detect_when_module_modifications_disabled - TypeError: expected str, bytes or os.PathLike obj...
FAILED test_command_sequencer.py::test_set_reload_to_true_when_no_modules_loaded - TypeError: expected str, bytes or os.PathLike object, not NoneType
FAILED test_command_sequencer.py::test_execute_sequence_with_missing_sequence - TypeError: expected str, bytes or os.PathLike object, not NoneType
FAILED test_command_sequencer.py::test_start_process_task - TypeError: expected str, bytes or os.PathLike object, not NoneType
FAILED test_command_sequencer.py::test_finish_process_task - TypeError: expected str, bytes or os.PathLike object, not NoneType
FAILED test_command_sequencer.py::test_finish_process_task_with_empty_process_tasks_list - TypeError: expected str, bytes or os.PathLike object, not NoneType
FAILED test_command_sequencer.py::test_add_context - TypeError: expected str, bytes or os.PathLike object, not NoneType
ERROR test_adapter.py::TestCommandSequenceManagerAdapter::test_get_valid_path - TypeError: expected str, bytes or os.PathLike object, not NoneType
ERROR test_adapter.py::TestCommandSequenceManagerAdapter::test_get_invalid_path - TypeError: expected str, bytes or os.PathLike object, not NoneType
ERROR test_adapter.py::TestCommandSequenceManagerAdapter::test_put_valid_path - TypeError: expected str, bytes or os.PathLike object, not NoneType
ERROR test_adapter.py::TestCommandSequenceManagerAdapter::test_put_invalid_path - TypeError: expected str, bytes or os.PathLike object, not NoneType
ERROR test_adapter.py::TestCommandSequenceManagerAdapter::test_put_bad_request - TypeError: expected str, bytes or os.PathLike object, not NoneType
ERROR test_adapter.py::TestCommandSequenceManagerAdapter::test_add_context - TypeError: expected str, bytes or os.PathLike object, not NoneType

```

--- a/src/odin_sequencer/manager.py
+++ b/src/odin_sequencer/manager.py
@@ -77,6 +77,9 @@ class CommandSequenceManager:
 
         for path in path_or_paths:
 
+            if not path:
+                continue
+
             if not isinstance(path, Path):
                 path = Path(path)
 

